### PR TITLE
Compose refactor

### PR DESCRIPTION
**Pull request by khawajaJunaid**
### What?
removes 'version' attribute post deprecation from all compose files related to dev, staging and prod env

### Why?
For the latest docker compose version i.e docker compose v2, the 'version' attribute has been deprecated, which is currently a warning when you do `docker compose up` but in future would become a requirement. https://github.com/compose-spec/compose-spec/blob/main/spec.md#version-and-name-top-level-elements this mentions that the version has become obsolete for the latest docker compose version. 

### How?
remove the version field in the compose files to remove the warning that one may get during builds
Closes NEXT-

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 1dfb4846fe1a5a0a934966fd299cd67f341a033a

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by khawajaJunaid** [comment]
hi @leerob, added the link to the documentation to the fact that the "version" attribute results in a warning as you asked, hope this helps

File: examples/with-docker-multi-env/docker/development/docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: "3"
+
 
 services:
```
Comment (leerob) [inline examples/with-docker-multi-env/docker/development/docker-compose.yml:3]:
```suggestion
services:
```

File: examples/with-docker-multi-env/docker/production/docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,3 @@
-version: "3"
 
 services:
```
Comment (leerob) [inline examples/with-docker-multi-env/docker/production/docker-compose.yml:2]:
```suggestion
services:
```

File: examples/with-docker-multi-env/docker/staging/docker-compose.yml [hunk 3]
```diff
@@ -1,4 +1,3 @@
-version: "3"
 
 services:
```
Comment (leerob) [inline examples/with-docker-multi-env/docker/staging/docker-compose.yml:2]:
```suggestion
services:
```

**Review by leerob — APPROVED** [review]

**Pull request merged**